tests/test_all.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+import requests
+import app
+from utils.tag_parsers import main_generic
+from utils.enrichment import enrich_test
+from tests.test_case_output import test_tag_parser_output, test_get_without_og_twitter_output, test_enrich_wikiart_artwork_output, test_enrich_imdb_tv_output, test_enrich_imdb_person_output, test_enrich_imdb_movie_output, test_enrich_goodreads_output
+
+
+def test_tag_parsers():
+    URL = 'https://www.youtube.com/watch?v=ab1H602yc_Y'
+    requested = requests.get(URL).text
+    get_data = main_generic(requested, URL)
+    assert get_data == test_tag_parser_output
+
+def test_get_without_og_twitter():
+    URL = 'https://oeis.org/wiki/List_of_LaTeX_mathematical_symbols'
+    requested = requests.get(URL).text
+    get_data = main_generic(requested, URL)
+    get_data['image'] = get_data['image'][:17] #hosting service generates a new url for every request
+    assert get_data == test_get_without_og_twitter_output
+
+def test_enrich_imdb_movie():
+    URL = 'https://www.imdb.com/title/tt0111161/?ref_=hm_fanfav_tt_9_pd_fp1'

```

Comment:
I like that we're testing this with the extra query params (`?rem_=...`)

We should ensure the parser returns a normalized, canonical URL without the extra query params